hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileRequest.java

```diff
@@ -226,7 +228,7 @@ public static OMPathInfoWithFSO verifyDirectoryKeysInPath(
         if (elements.hasNext()) {
```

Comment:
IMO, instead of changing PathInfo, it can filter Default at time of Directory / file creation. FileInfo creation, its handled. But directory creation for all flow, this needs handle.